src/graph_notebook/magics/graph_magic.py

```diff
@@ -163,37 +165,54 @@ def sparql(self, line='', cell='', local_ns: dict = None):
         args = parser.parse_args(line.split())
         mode = str_to_query_mode(args.query_mode)
         tab = widgets.Tab()
+        titles = []
+        children = []
+
+        metadata = {
+            "Query mode": args.query_mode,
+            "Total execution time (ms)": "N/A",
+            "# of results": "N/A",
+            "Response size (bytes)": "N/A",
+            "Results size (bytes)": "N/A"
+        }
 
         path = args.path if args.path != '' else self.graph_notebook_config.sparql.path
 
         logger.debug(f'using mode={mode}')
         if mode == QueryMode.EXPLAIN:
+            query_start = time.time() * 1000
```

Comment:
I'm guessing this makes `query_start` be measured in seconds? Can we add some documentation there? We probably want to report the latency of a query in milliseconds

Reply:
Query_start and query_time are in milliseconds; time.time() returns current time in seconds as a high precision float so I multiplied by 1000 to convert to ms. I'll add some comments in the code with this info